traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -324,6 +324,24 @@ WHERE ds.id = $1
 	return name, cdn, true, nil
 }
 
+// returns returns the delivery service name and cdn, whether it existed, and any error.
```

Comment:
nit but this GODOC should start with `GetDSIDAndCDNFromName` and it returns the ds id not name